src/libpgmoneta/wf_backup.c

```diff
@@ -414,56 +414,62 @@ basebackup_execute(char* name __attribute__((unused)), struct art* nodes)
    }
 
    pgmoneta_create_info(backup_base, label, 1);
```

Comment:
Is it necessary to explicitly create the info file? Can you simply define the struct, populate with all the fields and save directly?